From: Dirk Mueller <dmueller@suse.com>
Subject: Fix build of mach-omap2/drm.c
Patch-Mainline: Not yet

Fix build issue due to implicit declaration of GET_OMAP_REVISION

Signed-off-by: Dirk Mueller <dmueller@suse.com>

--- a/arch/arm/mach-omap2/drm.c.orig	2013-01-15 21:16:20.802729245 +0100
+++ b/arch/arm/mach-omap2/drm.c	2013-01-15 21:16:34.670580887 +0100
@@ -27,6 +27,7 @@
 
 #include "omap_device.h"
 #include "omap_hwmod.h"
+#include "soc.h"
 
 #if defined(CONFIG_DRM_OMAP) || (CONFIG_DRM_OMAP_MODULE)
 
